### PR TITLE
gpg plugin is added.

### PR DESCRIPTION
**Pull request by sumauppa**
gpg plugin is added to pom.xml.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -66,23 +66,23 @@
     <name>${extension.name} API</name>
     <description>Project GlassFish Enterprise JavaBean API</description>
     
-    <url>http://metro.java.net/</url>
+    <url>https://javaee.github.io/metro/</url>
```
Comment (yaminikb) [inline pom.xml]: This doesn't seem right, this is not a metro project

**Review by yaminikb — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (sumauppa) [reply pom.xml]: This does not have corresponding rpc project under javaee.github.io. Not sure, why did they use metro URL before. Shall I leave it to the old value?

Comment (yaminikb) [reply pom.xml]: Remove the url.

File: pom.xml [hunk 2]
```diff
@@ -64,13 +64,11 @@
...
-    <url>http://metro.java.net/</url>
-
     <organization>
-        <name>GlassFish Community</name>
-        <url>https://glassfish.java.net</url>
+        <name>Oracle Corporation</name>
+        <url>http://www.oracle.com</url>
     </organization>
     <licenses>
         <license>
```
Comment (yaminikb) [inline pom.xml:74]: License URL should be https://oss.oracle.com/licenses/CDDL+GPL-1.1

**Review by yaminikb — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (sumauppa) [reply pom.xml:74]: Done the change.

**Review by yaminikb — APPROVED** [review]

**Pull request merged**